ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -153,6 +154,24 @@ const witchHuntAlertOutputStrings = {
     cn: '引导远 (小队近)',
     ko: '멀리 유도 (본대 가까이)',
   },
+  baitNear: {
+    en: 'Close (Party Far)',
```

Comment:
I had originally intended to call "Close" for all of the options, but that doesn't work because of melee/ranged issues due to parties not being perfect composition. Makes sense in that case to just always call the role. ok.